GDSCommon-Demo/GDSCommon-DemoTests/BaseViewControllerTests.swift

```diff
@@ -0,0 +1,35 @@
+@testable import GDSCommon
+import GDSCommon_Demo
+import XCTest
+
+final class BaseViewControllerTests: XCTestCase {
+    var viewModel: BaseViewModel!
+    var sut: BaseViewController!
+    
+    override func setUp() {
+        super.setUp()
+        viewModel = MockBaseViewModel {
+            
+        } dismissAction: {
+            
+        }
+        sut = BaseViewController(viewModel: viewModel, nibName: "MockBase", bundle: .init(for: BaseViewControllerTests.self))
+    }
+    
+    override func tearDown() {
+        viewModel = nil
+        sut = nil
+        
+        super.tearDown()
+    }
+}
+
+extension BaseViewControllerTests {
```

Comment:
should we also have tests for the actions?

Reply:
tests added